lunabot_control/scripts/differential_drive_controller.py

```diff
@@ -73,40 +65,8 @@ def _loop(self):
         left_set = self.lin - self.ang * self.width / 2
         right_set = self.lin + self.ang * self.width / 2
 
-        # Measured Velocity in m / s
-        left_measured = self._left_vel * self._meters_per_rad
-        right_measured = self._right_vel * self._meters_per_rad
-
-        # Calculating error
-        left_error = left_set - left_measured
-        right_error = right_set - right_measured
-
-        self.left_error_sum += left_error
-        self.right_error_sum += right_error
-
-        self.left_error_sum = np.clip(self.left_error_sum, -self.i_sat, self.i_sat)
-        self.right_error_sum = np.clip(self.right_error_sum, -self.i_sat, self.i_sat)
-
-        # Calculating motor velocities
-        left = (
-            left_percent_estimate
-            + left_error * self.p
-            + self.left_error_sum * self.i
-            + (left_error - self._left_prev_error) / self.loop_dt * self.d
-        )
-        right = (
-            right_percent_estimate
-            + right_error * self.p
-            + self.right_error_sum * self.i
-            + (right_error - self._right_prev_error) / self.loop_dt * self.d
-        )
-
-        # Calculating previous error
-        self._left_prev_error = left_error
-        self._right_prev_error = right_error
-
-        left_drive_msg.data = self.constrain(left)
-        right_drive_msg.data = self.constrain(right)
+        left_drive_msg.data = self.constrain(left_set * self._meters_per_rad)
```

Comment:
This goes and uses the self._meters_per_rad. Due to github, I can't leave a comment directly on that definition, but I think that number may be wrong. Even if it is correct, something more than a magic number would be appreciated.